instructions/js/index.js

```diff
@@ -6,12 +6,54 @@ copyright.innerHTML = ` roselynn ${thisYear}`;
 footer.appendChild(copyright);
 
 const skills = ['Html,Css and Javascript','Drupal','Q|A with webdriversIO','Service Now'];
-let skillsSection = document.getElementById("#skills")
+let skillsSection = document.getElementById("skills")
 let skillList = skillsSection.querySelector('ul') //grabs ul element from #skills 
 
 for(let i = 0; i < skills.length; i ++){ 
     let skill = document.createElement("li")
     skill.innerText = skills[i];
     skillList.appendChild(skill)
 
-}
\ No newline at end of file
+}
+
+//Handle Message Form Submit
+const messageForm = document.querySelector('form[name="leave_message"]');
+messageForm.addEventListener("submit", myFunction);
+
+//creating callback function
+function myFunction(){
+event.preventDefault();
+const eventTarget = event.target ;//event is the submit;  target is the form.
+const firstInput = eventTarget.querySelector('input[name="name"]');
+console.log(firstInput.value);
+const secondInput = eventTarget.querySelector('input[type="email"]');
+console.log(secondInput.value);
+const thirdInput = eventTarget.querySelector('#message');
+console.log(thirdInput.value);
+
+
+//Display message in List
+const messageSection = document.getElementById("messages");
+const messageList = messageSection.querySelector('ul');
+const newMessage = document.createElement("li");
+newMessage.innerHTML = `<a href="mailto: ${secondInput}">${firstInput}</a> wrote:  <span>${thirdInput}</span>`;
```

Comment:
Looks pretty good, I think this line is the only issue. When you use the variables `firstInput`, `secondInput`, etc. you're referencing the input itself rather than the text that the input contains.